### ISSUE TITLE
Dutch tokenizer doesn't split numbers from units

### ISSUE THREAD

**Issue by iliakur** [explosion/spaCy#8281]
Thanks in advance for your attention. And thanks for the library, we use it heavily every day :)

## How to reproduce the behaviour

```python
>>> import nl_core_news_md
>>> nlp = nl_core_news_md.load(disable=["ner"])
>>> [tok.text for tok in nlp.tokenizer("21%")]
["21%"]
```

The equivalent setup and input for German splits the number from the percentage:
```python
>>> import de_core_news_md
>>> nlp = de_core_news_md.load(disable=["ner"])
>>> [tok.text for tok in nlp.tokenizer("21%")]
["21", "%"]
```

Comparing the suffix rules for the two languages I noticed that German has this regex:
```python
r"(?<=[0-9])(?:km|km²|km³|m|m²|m³|dm|dm²|dm³|cm|cm²|cm³|mm|mm²|mm³|ha|µm|nm|yd|in|ft|kg|g|mg|µg|t|lb|oz|m/s|km/h|kmh|mph|hPa|Pa|mbar|mb|MB|kb|KB|gb|GB|tb|TB|T|G|M|K|%|км|км²|км³|м|м²|м³|дм|дм²|дм³|см|см²|см³|мм|мм²|мм³|нм|кг|г|мг|м/с|км/ч|кПа|Па|мбар|Кб|КБ|кб|Мб|МБ|мб|Гб|ГБ|гб|Тб|ТБ|тбكم|كم²|كم³|م|م²|م³|سم|سم²|سم³|مم|مم²|مم³|كم|غرام|جرام|جم|كغ|ملغ|كوب|اكواب)"
```

I'm no native speaker of Dutch, but it seems reasonable to add this regex to the tokenizer just like in German. Also fine if it's a no-go, at least we know for sure and can document this customization in our pipeline setup.

## Your Environment

- **spaCy version:** 3.0.6
- **Platform:** Linux-5.12.6-200.fc33.x86_64-x86_64-with-fedora-33-Thirty_Three
- **Python version:** 3.7.9
- **Pipelines:** de_core_news_md (3.0.0), en_core_web_sm (3.0.0), nl_core_news_md (3.0.0), it_rtrlaser_3 (0.1.2), fr_core_news_lg (3.0.0)

**Comment by svlandeg** [comment]
Hm, this is interesting. Most languages indeed have the following suffix rule:
```
r"(?<=[0-9])(?:{u})".format(u=_units),
```

But Spanish and Dutch specifically exclude the `%` in that `_units` list, cf https://github.com/explosion/spaCy/blob/master/spacy/lang/nl/punctuation.py#L30

I kind of see the case for keeping it as one token: in the end, "0.30" is the same as "30%" and can be seen as one token of information. 

Additionally, it looks like @adrianeboyd specifically added this exception for `%` based on corpus statistics: https://github.com/explosion/spaCy/pull/5259

So in short, I think we probably want to keep the tokenization as is, but do note that other suffixes like `km³` will in fact be split off, just like in German.

**Comment by iliakur** [comment]
Thanks, good to know! For our usecase we're likely gonna have to keep the %-splitting behavior, so we'll customize the rule accordingly.

**Comment by adrianeboyd** [comment]
Yes, most of the tokenizer settings are based on the training data used for the pretrained pipelines like `nl_core_news_sm`. If it's not the right setting for your data/task, customizing the tokenizer is the right way to go.
